ci/run_premerge_gpu.sh

```diff
@@ -75,24 +77,27 @@ verify_bundle() {
             python $(pwd)/ci/prepare_schema.py --l "$bundle_list"
         for bundle in $bundle_list;
         do
-            # Check if the bundle is "maisi_ct_generative", if so, set local environment (venv cannot work with xformers)
-            if [ "$bundle" == "maisi_ct_generative" ]; then
-                echo "Special handling for maisi_ct_generative bundle"
-                set_local_env requirements-dev.txt
-            else
-                init_venv requirements-dev.txt
-            fi
             # get required libraries according to the bundle's metadata file
-            requirements=$(python $(pwd)/ci/get_bundle_requirements.py --b "$bundle")
+            requirements_file="requirements_$bundle.txt"
+            python $(pwd)/ci/get_bundle_requirements.py --b "$bundle" --requirements_file "$requirements_file"
             # check if ALLOW_MONAI_RC is set to 1, if so, append --pre to the pip install command
             if [ $ALLOW_MONAI_RC = true ]; then
                 include_pre_release="--pre"
             else
                 include_pre_release=""
             fi
-            if [ ! -z "$requirements" ]; then
+            # Set a new environment to verify bundle
+            if [ "$bundle" == "maisi_ct_generative" ]; then
```

Comment:
Why make MAISI specific logic?

Thanks.

Reply:
Hi @Nic-Ma , as mentioned in the line below, we use venv to setup environment for each bundle, but maisi requires `xformers` which cannot work with it

Reply:
Hi @mingxin-zheng , I remember you know something about the `xformers` installation? Could you please help review this PR?

Thanks.

Reply:
The code change does not impact the maisi's logic, and I only adjust the order to ensure:
1) the container environment is used to get requirements for each bundle
2) a new venv is used to verify each bundle
cc @Nic-Ma @mingxin-zheng

Reply:
I remember MAISI doesn't require `xformers` anymore in version before the latest release, but I could be wrong.

Reply:
Thanks for the info @mingxin-zheng . The current method is implemented when we involved the initial MAISI, maybe now `xformers` is not necessary. Let me check it

Reply:
updated